spec/models/course/assessment/answer/scribing_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe Course::Assessment::Answer::Scribing, type: :model do
+  it { is_expected.to act_as(Course::Assessment::Answer) }
+  it 'has many scribbles' do
+    expect(subject).to have_many(:scribbles).
+      class_name(Course::Assessment::Answer::ScribingScribble.name).
+      dependent(:destroy)
+  end
+
+  let(:instance) { Instance.default }
+  with_tenant(:instance) do
+    describe '#reset_answer' do
+      let(:answer) { create(:course_assessment_answer_scribing) }
+      subject { answer.reset_answer }
+
+      it 'removes all scribbles' do
+        expect(subject.specific.content).to eq('')
```

Comment:
Can you modify the expect parameter to have `subject.specific.reload.content` to just make sure the change is persisted?